strategies/changed/src/main/java/org/arquillian/smart/testing/vcs/git/GitChangesDetector.java

```diff
@@ -75,7 +93,7 @@ private File findRepoRoot(File currentDir) {
             .map(diffEntry -> {
                 try {
                     final File sourceFile = new File(repoRoot, diffEntry.getNewPath());
-                    return new ClassNameExtractor().extractFullyQualifiedName(sourceFile);
+                    return ClassNameExtractor.extractFullyQualifiedName(sourceFile);
```

Comment:
Sure, I completely agree with you, just that when I was reviewing this, I noticed this thing.